types/react-dragula/tsconfig.json

```
@@ -0,0 +1,26 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "jsx": "react",
+        "esModuleInterop": true
```

```suggestion
        "jsx": "react"
```
This is not advised to be used in DT package itself, unless there is a solid reason (e.g. 3rd party library cannot be correctly imported for tests or reference in DT package). This is for consumers of the DT package itself.
Your DT will compile and pass tests with that change (and reworked imports in tests) just fine.

---

That's great, thanks a bunch for the review and suggestions!